src/Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+
```

Comment:
@andreasohlund @bording if we exclude this file we can have reposync enabled I think.